### PR TITLE
Add --force option to tag script (#108)

### PR DESCRIPTION
**Pull request by pypaut**
This is my first ever pull request to an open-source repository, please do not hesitate to give feedback.

### CODE REVIEW

**Comment by ortex** [comment]
Hi, thanks for PR!

Could you please attach the console log or/and screenshots of how you tested it?

**Comment by pypaut** [comment]
Hello, I had tested on a different repository, did it again more thoroughly on roc-go.
Here are my logs: https://privatebin.net/?16578bd480ae82e8#EgzUM9TJKCXdSByAyx1bv9duYcrAbKnLjK3TBDcf9pff

File: tag.py [hunk 1]
```diff
@@ -28,23 +28,38 @@ def run_command(args):
     print(f'--- Running command: {" ".join(args)}')
     subprocess.check_call(args)
 
-def commit_change(version):
+def commit_change(version, force):
     run_command(['git', 'add', 'roc/version.go'])
-    run_command(['git', 'commit', '-m', f'Release {version}'])
+    try:
+        run_command(['git', 'commit', '-m', f'Release {version}'])
+    except subprocess.CalledProcessError:
```
Comment (ortex) [inline tag.py:35]: I think it's better to check if we have something to commit instead of catching a general error here

Comment (pypaut) [reply tag.py]: Done! It feels better this way, thank you.

I did the same tests as in the previous privatebin.

Comment (gavv) [reply tag.py]: Small suggestion: we could first run "git add", and only then "git diff", but with "--cached" option. This way we will handle all 3 situations: when roc/version.go is unchanged, changed and not staged, or changed and staged.

**Review by gavv — APPROVED** [review]

File: tag.py [hunk 2]
```diff
@@ -28,23 +28,39 @@ def run_command(args):
     print(f'--- Running command: {" ".join(args)}')
     subprocess.check_call(args)
 
-def commit_change(version):
+def check_version_to_commit():
+    cmd = ['git', 'diff', '--cached', 'roc/version.go']
+    return subprocess.check_output(cmd) != b''
+
+def commit_change(version, force):
```
Comment (ortex) [inline tag.py:35]: nit: `force` arg can be removed now

**Review by ortex — APPROVED** [review]

**Comment by ortex** [comment]
Thanks @pypaut !

We have a similar script in another repo https://github.com/roc-streaming/roc-droid/blob/main/tag.py it would be nice to update it as well 😄

**Pull request merged**